src/lib/cli/command.js

```diff
@@ -36,8 +36,15 @@ let _opts = {};
 
 let alreadyConfirmedDebugAttachment = false;
 
+/**
+ * @param {string[]} argv
+ */
 // eslint-disable-next-line complexity
 args.argv = async function ( argv, cb ) {
+	if ( process.platform !== 'win32' && argv[ 1 ]?.endsWith( '.js' ) ) {
```

Comment:
`args` has a special treatment for Windows:

```js
      const binaryExt = path.extname(this.binary)
      const mainModule = process.env.APPVEYOR
        ? '_fixture'
        : process.mainModule.filename
      
      full = `${mainModule}-${subCommand}`
      
      if (path.extname(this.binary)) {
        full = `${mainModule.replace(binaryExt, '')}-${subCommand}${binaryExt}`
      }
```

We must not strip `.js` from the script name so that `args` can construct the script's name properly.